ethcore/evm/src/interpreter/shared_cache.rs

```diff
@@ -28,8 +28,8 @@ const DEFAULT_CACHE_SIZE: usize = 4 * 1024 * 1024;
 // stub for a HeapSizeOf implementation.
```

Comment:
This comment is interesting. Is the `Bit` type there just to impl `HeapSizeOf`? If yes, do we still need it? 

Either way the comment needs updating. ;)